ci/cirrus_wheels.yml

```diff
@@ -24,7 +24,7 @@ cirrus_wheels_linux_aarch64_task:
     # single task takes longer than 60 mins (the default time limit for a
     # cirrus-ci task).
     - env:
-        CIBW_BUILD: cp310-manylinux*
+        CIBW_BUILD: cp310-manylinux* cp313-manylinux*
```

Comment:
I think that below, where we have `pip install numpy>=2.0.0.dev0`, we'll need an adjustment for Python `3.13.0rc1`. Well, Chuck may be about to release a stable version of `3.13`-compatible NumPy `2.1.0` soon, but at the moment I can make more progress on a Linux `aarch64` Python `3.13.0rc1` setup when I change that to `pip install numpy==2.1.0rc1` (though we likely only want that, or a similar way to get at least that pre-release, for `3.13.0rc1`).